assert.js

```diff
@@ -0,0 +1,21 @@
+/**
+  * Ensures an expected string is not empty.
+  *
+  * @param {Object} options - Options to use.
+  * @param {string} options.value - The expected string.
+  * @param {string} options.key - The identifier for the parameter.
+  *
+  * @throws - If the value is not a string or is empty.
+  * @returns {undefined} - It just throws or you are ok.
+*/
+export function notEmptyString({value, key}) {
```

Comment:
I think I prefer the syntax that `assert-plus` has: https://www.npmjs.com/package/assert-plus

I'm OK if others are comfortable with this.

If named parameters are preferred, I think something other than value and key would be good.

Reply:
I do agree with you here, I like the way assert-plus handles this so changed them to match assert-plus here: https://github.com/digitalbazaar/bedrock-web-profile-manager/pull/26/commits/95811573ea9afb72d6b988a00aea76f44afe5446